packages/features/bookings/Booker/Booker.tsx

```diff
@@ -72,9 +75,20 @@ const BookerComponent = ({
     shallow
   );
 
+  const date = dayjs(selectedDate).format("YYYY-MM-DD");
+  const schedule = useScheduleForEvent({prefetchNextMonth: true});
+  const nonEmptyScheduleDays = useNonEmptyScheduleDays(schedule?.data?.slots).filter((slot)=>dayjs(selectedDate).diff(slot,'day')<=0);;
+  
   const extraDays = isTablet ? extraDaysConfig[layout].tablet : extraDaysConfig[layout].desktop;
   const bookerLayouts = event.data?.profile?.bookerLayouts || defaultBookerLayoutSettings;
   const animationScope = useBookerResizeAnimation(layout, bookerState);
+  const addonDays = nonEmptyScheduleDays.length < extraDays ? (extraDays - nonEmptyScheduleDays.length + 1) * 7  : 0;
+  const availableSlots = nonEmptyScheduleDays.slice(0,  extraDays + 1);
+  if(nonEmptyScheduleDays.length !==0 )
+    newExtraDays.current =  Math.abs(dayjs(selectedDate).diff(availableSlots[availableSlots.length - 2],'day')) + addonDays;
+  const prefetchNextMonth = dayjs(date).month() !== dayjs(date).add(newExtraDays.current, "day").month();
+  const monthCount = dayjs(date).add(1,"month").month() !== dayjs(date).add(newExtraDays.current, "day").month() ? 2 : undefined
+  const nextslots = Math.abs(dayjs(selectedDate).diff(availableSlots[availableSlots.length - 1],'day')) + addonDays;
```

Comment:
```suggestion
  const nextSlots = Math.abs(dayjs(selectedDate).diff(availableSlots[availableSlots.length - 1],'day')) + addonDays;
```
Following name convention

Reply:
need to use same name in `Header.tsx` too. No issues I'll do ✌️